package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "micromark",
+  "version": "0.0.1",
```

Comment:
We have to start at `2.0.0` as we got the `micromark` package from someone who has already published a `1.0.0`.

Reply:
If we’d use the scope, `@micromark`, we can start at 1.0.0. If we don’t scope, we need to start above 2.0.0. I’d say leave it at this and we can decide later.